operatorconfig/driverconfig/powerflex/v2.13.0/node.yaml

```diff
@@ -30,6 +30,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["csinodes"]
+    verbs: ["get", "list", "watch", "update"]
```

Comment:
Are watch and update really required for the mdm-container to read node labels?

Reply:
this isn't part of mdm-container.

I saw an unrelated error in driver logs about not being able to watch CSInodes object. 
We must have had this error in our driver for awhile now.